core_backend/app/auth/routers.py

```diff
@@ -240,6 +240,7 @@ async def authenticate_or_create_google_user(
             api_daily_quota=DEFAULT_API_QUOTA,
             asession=asession,
             content_quota=DEFAULT_CONTENT_QUOTA,
+            doc_language=DEFAULT_DOC_LANGUAGE,
```

Comment:
When a new user registers via google credentials, their workspace document language will initially be `ENGLISH`. The user can then update this value in the UI after logging in.